projects/cdk/tokens/fonts-ready.ts

```diff
@@ -0,0 +1,14 @@
+import {DOCUMENT} from '@angular/common';
+import {inject, InjectionToken} from '@angular/core';
+import {from, NEVER, Observable} from 'rxjs';
+import {shareReplay} from 'rxjs/operators';
+
+export const TUI_FONTS_READY = new InjectionToken<Observable<unknown>>(
+    `[TUI_FONTS_READY]`,
+    {
+        factory: () =>
+            from((inject(DOCUMENT) as any).fonts?.ready || NEVER).pipe(
```

Comment:
Guys, `NEVER` or `EMPTY` ? 🤔

Reply:
I think `EMPTY` prefer here

NEVER well… It never fires anything. It doesn’t emit anything nor does it complete the observable. This may be useful in very limited cases where you need to return a value, but you don’t want subscribers to act on it *or* for anyone to act on the observable completion. (c)

Reply:
I think `EMPTY`, because it terminates, and `NEVER` does not

Reply:
fixed